components/fieldset/src/test.tsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import { render } from '@testing-library/react';
 
 import { FieldsetDocumented as Fieldset } from '.';
 
 describe('Fieldset', () => {
-  it('matches snapshot', () => {
-    expect(mount(<Fieldset />)).toMatchSnapshot('Fieldset');
+  it('renders contents without crashing', () => {
```

Comment:
from the perspective of the user, I think this test would read 'contents is visible'